temporallib/client/client.py

```diff
@@ -28,6 +30,11 @@ class Options:
     tls_root_cas: str = None
     auth: AuthOptions = None
 
+async def update_rpc_metadata(client_opt, rpc_metadata):
+    while True:
+        await asyncio.sleep(3300)  # Refresh every 55 minutes
```

Comment:
55 minutes seems arbitrary, any specific reasoning? If so, it might be nice to put it in a comment. Also, would it be unnecessary to put this in a constant? Any chance it will be used elsewhere?

Reply:
I parameterized the interval value and added a comment explaining the logic behind 55 minutes, which is to accommodate for the Google Oauth token 60 minute expiry time.